docs/overview/cmake-can-not.rst

```diff
@@ -119,30 +119,27 @@ platform independently.
 Unrelocatable projects
 ======================
 
-Internally :ref:`CMake <CMake>` save full paths to the sources so it's not
-possible to generate project and then share it between several developers.
-It means you can't be CMake-guy who will generate different projects for
-Xcode-guys and VisualStudio-guys. All developers in team at least should be
-aware of how to generate projects using CMake. On practice it means they have
-to know CMake arguments to use, literally it's
-``cmake -H. -B_builds -GXcode``/``cmake -H. -B_builds "-GVisual Studio 12 2013"``,
-plus remember workflow notes from the section above.
-Next logic can be applied here: if you're working with developer then developer
```

Comment:
I think this part is important, please keep it.

Reply:
I agree that it is important; however, I do not think that it belongs where it is. It is sufficient to state that CMake must be run on each of the machines that will utilize the generated files, rather than having a single computer generate them for everyone. Including the additional admonitions on what developers should do is overly pedagogical and would be more suited to a good practice section. I understand that you are trying to include such good practices as examples in this document, but assaulting the reader so early on can hardly be called helpful.

I have re-added the content, but reworded it so it reads more clearly.

Reply:
> is overly pedagogical and would be more suited to a good practice section

> assaulting the reader so early on can hardly be called helpful

I mostly have tried to provide "problem -> solution" pattern. It fit the statement in the beginning:
```
Most of the drawbacks mentioned here can be workarounded by using approaches
that may differs from your normal workflow but reach the end goal. Try to look
at them from another angle, think of the picture in a whole and remember that
advantages definitely outweigh the disadvantages.
```